### PR TITLE
Database implementation and seeding script set up

### PR DESCRIPTION
**Pull request by swalk001**

### CODE REVIEW

File: server/index.js [hunk 1]
```diff
@@ -1,9 +1,13 @@
 const express = require('express');
 const app = express();
 const port = process.env.PORT || 3001;
+const db = require('../db/index.js')
 
 app.use(express.static(__dirname + '/../public/dist'))
 
+
```
Comment (james-b-olivas) [inline server/index.js]: A lot of whitespace here - I don't think that this is necessary.

File: db/index.js [hunk 2]
```diff
@@ -1,7 +1,15 @@
 const mongoose = require('mongoose');
-mongoose.connect('mongodb://localhost/listing_images');
+mongoose.connect('mongodb://localhost/listing_images', { useUnifiedTopology: true, useNewUrlParser: true });
```
Comment (james-b-olivas) [inline db/index.js]: As far as I'm aware, there's an 80-character per line limit on the style guide. To avert this, I would recommend displaying the settings object across multiple lines, or by assigning the settings to a variable.

File: db/seeding.js [hunk 3]
```diff
@@ -0,0 +1,23 @@
+const db = require('./index.js')
+
+let constURL = "https://acoustic-guitar-images.s3.us-east-2.amazonaws.com/"
+
+let generateDB = () => {
+ db.dropDatabase();
+  let id = 1;
+  let listingId = 0;
+  for (var i = 1; i < 501; i++) {
+    if((i-1)%5 === 0){
```
Comment (james-b-olivas) [inline db/seeding.js]: There should be a space between the closing brace and the opening bracket.

**Review by james-b-olivas — CHANGES_REQUESTED** [review]
Hi, Steven! Overall, I think this is pretty good, but I caught some styling errors that I think ought to be fixed.

**Review by swalk001 — COMMENTED** [review]
Thanks, James. Appreciate it!

**Pull request merged**